### PR TITLE
fix(rosco): Update Hashicorp Packer version in rosco to 1.6.6

### PR DESCRIPTION
**Pull request by EasyAsABC123**
We prefer small, well tested pull requests.

Please refer to [Contributing to Spinnaker](https://spinnaker.io/community/contributing/).

When filling out a pull request, please consider the following:

* Follow the commit message conventions [found here](http://www.spinnaker.io/v1.0/docs/how-to-submit-a-patch).
* Provide a descriptive summary for your changes.
* If it fixes a bug or resolves a feature request, be sure to link to that issue.
* Add inline code comments to changes that might not be obvious.
* Squash your commits as you keep adding changes.
* Add a comment to @spinnaker/reviewers for review if your issue has been outstanding for more than 3 days.

Note that we are unlikely to accept pull requests that add features without prior discussion. The best way to propose a feature is to open an issue first and discuss your ideas there before implementing them.

### CODE REVIEW

File: Dockerfile.java8 [hunk 1]
```diff
@@ -6,10 +6,10 @@ COPY halconfig/packer              /opt/rosco/config/packer
 
 WORKDIR /packer
 
-RUN apk --no-cache add --update bash wget curl openssl openjdk8-jre && \
-  wget https://releases.hashicorp.com/packer/1.4.4/packer_1.4.4_linux_amd64.zip && \
-  unzip packer_1.4.4_linux_amd64.zip && \
-  rm packer_1.4.4_linux_amd64.zip
+RUN apk --no-cache add --update bash wget curl openssl openjdk8-jre git openssh-client && \
```
Comment (drewhodson) [inline Dockerfile.java8:9]: Just curious, why the addition of `git` and `openssh-client`?

Comment (EasyAsABC123) [reply Dockerfile.java8:9]: @drewhodson i'm guessing this is due to a merge from upstream spinnaker i'll remove this if you desire

**Review by kfrank2017 — APPROVED** [review]
looks alright to me.

File: Dockerfile.java8 (hunk 1 above)
Comment (drewhodson) [reply Dockerfile.java8:9]: I'm fine with keeping it, just curious 🙂

**Review by drewhodson — APPROVED** [review]

**Review by ben-lindsey — APPROVED** [review]

**Pull request merged**